pkg/security/probe/process_killer.go

```diff
@@ -387,6 +486,34 @@ func (p *ProcessKiller) Start(ctx context.Context, wg *sync.WaitGroup) {
 				}
 			case running:
 				select {
+				case nextKillAlarm := <-p.setKillQueue:
+					if killQueueAlarm != nil {
+						killQueueAlarm.Stop()
+					}
+					killQueueAlarm = time.NewTicker(time.Until(nextKillAlarm))
+					killQueueChan = killQueueAlarm.C // enable the ticker by setting its channel
+					break
+
+				case <-p.disableKillQueue:
+					if killQueueAlarm != nil {
+						killQueueAlarm.Stop()
+						killQueueAlarm = nil
+					}
+					killQueueChan = nil
+					p.currentKillQueueAlarm = nil
+					break
+
+				case <-killQueueChan:
```

Comment:
would it possible to use only one ticker let say 250ms call the killqueue function ? it will avoid having 3 chan and most of the time the queue list will be empty.

Reply:
Yeah .. I looked to have a start/stop alarm but at the end the implementation is a bit too complex for nothing I guess, I'll update it with a single chan

Reply:
updated